src/app/modal/modal.demo.routing.ts

```diff
@@ -34,6 +35,7 @@ const ROUTES: Routes = [
             { path: "backdrop", component: ModalBackdropDemo },
             { path: "sizes", component: ModalSizesDemo },
             { path: "max-height", component: ModalMaxHeightDemo }
```

Comment:
@victormejia 
TSLint is complaining about a missing comma at the end of line 37 here.